### PR TITLE
New api methods for aa

### PR DESCRIPTION
**Pull request by Taump**
Changes proposed in this pull request:
- add method getAasByBaseAss
- add method getAaResponses

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -239,6 +239,22 @@ declare namespace Obyte {
         callback?: (err: null | string, result: string) => void,
       ): Promise<object>;
 
+      getAasByBaseAss(
```
Comment (bonustrack) [inline index.d.ts]: Some typo here :) 
```suggestion
      getAasByBaseAas(
```

File: index.d.ts [hunk 2]
```diff
@@ -239,6 +239,22 @@ declare namespace Obyte {
...
+        params: {
+          base_aa: object;
```
Comment (bonustrack) [inline index.d.ts]: The type seem wrong, this should be a string and be optional
```suggestion
          base_aa?: string;
```
Same for `base_aa` and `aa`, `aas` on `getAaResponses`

File: src/api.json [hunk 3]
```diff
@@ -93,6 +93,18 @@
       "address": "string"
     }
   },
+  "light/get_aas_by_base_aas": {
+    "params": {
+      "base_aa": "object",
```
Comment (bonustrack) [inline src/api.json]: Here and the others `base_aas`, `aa` and `aas` also should be changed to string like this: 
```suggestion
      "base_aa": "string",
```

File: index.d.ts (hunk 1 above)
Comment (Taump) [reply index.d.ts]: :D

**Pull request merged**